extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -504,9 +505,9 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
   /** @override */
   shouldPreferentialRenderWithoutCrypto() {
-    return experimentFeatureEnabled(
-        this.win, DOUBLECLICK_EXPERIMENT_FEATURE.CANONICAL_EXPERIMENT,
-        DFP_CANONICAL_FF_EXPERIMENT_NAME);
+    return !isCdnProxy(this.win) && !experimentFeatureEnabled(
```

Comment:
Is this CDN check actually necessary?  If on the CDN, then page is SSL so crypto should be present.  How about instead adding dev().assert(!isCdnProxy(this.win));

Reply:
done